artwork.md

```diff
@@ -7,10 +7,12 @@ permalink: /artwork/
 This Page is for displaying SS3D-related artworks such as wallpapers, animations and music created by artists of the community.
 
 # Brukas
+{: class="centred" }
```

Comment:
Might be better to make all headers centred in the CSS, if that's what we actually want?

Reply:
I tried this originally but couldn't figure out how to make that happen for only the artwork page in the short time I looked at it. So this was my short term solution, but if you can point me in the right direction on how to make it for artwork page only, I can try to do so... it seems I might need a new layout.html?

Reply:
Yeah that would be correct unfortunately. I wouldn't object to you doing it this way for now as long as we try to fix it in the future

Reply:
Yeah, I would also prefer it that way as well but it's currently beyond my knowledge and time.